auth/feature-tests/functional/fixtures/auth.fixtures.ts

```diff
@@ -1,4 +1,9 @@
 export const AuthFixtures = {
+    valid: {
+        username: 'test@test.com',
+        password: 'Admin123!', // pragma: allowlist-secret 
```

Comment:
As mentioned previously, this is not a sustainable long-term approach and should be stored in secrets or created via the [admin sdk](https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_AdminCreateUser.html)